test($sanitize): skip clobber test on Edge

Narretz opened the pull request:
<!-- General PR submission guidelines https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#submit-pr -->
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**



**What is the current behavior? (You can also link to an open issue here)**



**What is the new behavior (if this is a feature change)?**



**Does this PR introduce a breaking change?**



**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our [guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- [ ] Fix/Feature: [Docs](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#documentation) have been added/updated
- [ ] Fix/Feature: Tests have been added; existing tests pass

**Other information**:

Narretz committed:
test($sanitize): skip nextSibling clobber test on Edge 16

In some circumstances, Edge 16 will not throw on clobbered
elements, because the nextSibling property is null. The exact
cause is currently unknown.

petebacondarwin commented on test/ngSanitize/sanitizeSpec.js:
```diff
@@ -246,25 +246,28 @@ describe('HTML', function() {
-        $sanitize('<form><input name="parentNode" /></form>');
-      }).toThrowMinErr('$sanitize', 'elclob');
+  if (!/Edge\/16/.test(window.navigator.userAgent)) {
+    // Skip test on Edge 16 due to browser bug.
```
Perhaps include this link here: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/15263856/#pagenotfound

Narretz replied:
Problem is that the issue is private, so it goes to 404. Maybe we should create an issue in our repo with a short description?

petebacondarwin reviewed (commented):
It just occurred to me that on Edge, when this fails to fail, the return value from `$sanitize` appears to be something like the console object.
Am I right?
How about we change the code so that if the result that will be returned from `$sanitize` is not a string then we throw the `elclob` error?
That way we don’t need to change our tests and we are covered from returning something dodgy in Edge.

mgol reviewed (approved):
`parentNode` is buggy in Edge as well but if our tests pass then I guess we may leave it as it is. LGTM.

mgol reviewed (approved):
LGTM but the `config.browserNoActivityTimeout` change should perhaps land in a separate commit.

Narretz committed:
chore(node): update jasmine to 2.8.0, and adjust tests

Narretz committed:
chore(node): update karma to 2.0.0

Also reduce karma log level on Travis to INFO.
Before, the log level was DEBUG, but it seems that
prior to karma 2.0.0, the debug messages were not outoput on Karma,
so this simply restores the status quo (and prevents cluttering the log).
